sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalog.scala

```diff
@@ -283,10 +294,15 @@ class SessionCatalog(
    * explicitly specified.
    */
   def isTemporaryTable(name: TableIdentifier): Boolean = {
-    !name.database.isDefined && tempTables.contains(formatTableName(name.table))
+    name.database.isEmpty && tempTables.contains(formatTableName(name.table))
   }
 
   /**
+   * Return whether View is supported
+   */
+  def isViewSupported: Boolean = false
```

Comment:
Yeah. You are right. 

If we remove it, we could drop a table in `SQLContext` by using a `DROP VIEW` command. I think that is the only issue we need to resolve.